cities/schemas.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str | None = None
+
+
+class CityCreate(CityBase):
+    pass
+
+
+class CityUpdate(BaseModel):
+    additional_info: str
+
+
+class City(CityBase):
+    id: int
+
+    class Config:
+        from_attributes = True
```

Comment:
The `from_attributes` option is not a standard Pydantic configuration option. If you intended to use `orm_mode`, which allows Pydantic models to be created from ORM objects, use `orm_mode = True` instead.